src/sentry_slack/plugin.py

```diff
@@ -140,12 +155,18 @@ def notify(self, notification):
                 })
 
         if self.get_option('include_tags', project):
+            included_tags = self.get_tag_list('included_tag_keys', project)
+            excluded_tags = self.get_tag_list('excluded_tag_keys', project)
             for tag_key, tag_value in self._get_tags(event):
-                fields.append({
-                    'title': tag_key.encode('utf-8'),
-                    'value': tag_value.encode('utf-8'),
-                    'short': True,
-                })
+                key = tag_key.lower()
```

Comment:
we saw this issue in rules, but maybe we should also handle the magical internal tags

i.e. release => sentry:release

This is our logic atm, but we repeat this code everywhere (which is awful for plugins):

```
        if obj.key.startswith('sentry:'):
            key = obj.key.split('sentry:', 1)[-1]
        else:
            key = obj.key
```

I'm generally ok with this change as is, just expanding on feedback here since its relevant.